project/maven-resolver.xml

```diff
@@ -3,14 +3,15 @@
     <tag>ASL 2.0</tag>
   </licensing>
   <module>
-    <dependency>slf4j</dependency>
```

Comment:
Is changing the order of dependencies intentional? If yes, what is the reason? If not, can we revert it?

Reply:
No, it is not, i thought it would be better to keep them in alphabetic order. I can revert it.